docs/rules/no-magic-numbers.md

```diff
@@ -128,6 +128,20 @@ a = data[4294967295]; // above the max array index
 a = data[1e500]; // same as data["Infinity"]
 ```
 
+### ignoreDefaultValues
+
+A boolean to specify if numbers used in default value assignments are considered okay. `false` by default.
+
+Examples of **correct** code for the `{ "ignoreDefaultValues": true }` option:
+
+```js
+/*eslint no-magic-numbers: ["error", { "ignoreDefaultValues": true }]*/
+
+const { tax = 0.25 } = accountancy;
+
+function mapParallel(concurrency = 3) { /***/ }
+```
```

Comment:
Can we add an example with a default assignment that isn't in a declaration?

It could be in an array patern `[... = ...] = ...;`

Reply:
I had a hard time coming up with a sensible example 🙂 hope this is fine. Otherwise please suggest anything else.

Reply:
The example looks great, just one small note: since it's in another block of code, we should add the `/* eslint ... ` comment there, too. Or, just add the example to the previous code block instead of creating the new one.

Reply:
Done.

Reply:
Thanks! Can we insert a blank line between the eslint comment and `let head;`?

Everything else looks good!